bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonDeviceHandler.java

```diff
@@ -373,7 +375,9 @@ public void initialize() {
                 });
 
                 if (ThingStatus.ONLINE == bridge.getStatus()) {
-                    updateStatus(ThingStatus.ONLINE);
+                    if (deviceLinked) {
```

Comment:
If device is not linked, is it intentional that status is not updated?

Reply:
The deviceNotLinked() method below will set the status and also set deviceLinked to false.  This code will only come into play when a user manually adds a device and enters the wrong address.